stable/kubernetes-dashboard/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: kubernetes-dashboard
-version: 1.2.2
+version: 1.2.3
```

Comment:
new feature. should  be at least 1.3.0.

Reply:
Got it

Reply:
Set to 1.3.0, not the tests can't be passed